### DISCUSSION TITLE
Global title

### DISCUSSION THREAD

**Discussion by thoath2** [kinghat/tabbed-card#148] [Feature Requests]
Hello,
is it possible to configure a global title/label of the card. The global title should placed over the tabs.
Thank you

**Comment by ildar170975** [comment]
As a workaround - place this card into smth like `stack-in-card` which supports a `title` option.

**Comment by thoath2** [comment]
Yes, the workaround works. Thanks for that. But I'm not really happy yet. Why not? In Figure 1 you can see that the tabs and the heading float quite in the air and apparently have no optical connection. The tabs are also not really taken as a tab. It would be nice if the tabs were actually part of the actually card body. Then you could also use the workaround.
The element "title" is also somewhat unfortunate. I can't switch it off. If I leave him empty, an unsightly gap to the new section (see Fig. 2)
Bild 1 


<img width="386" alt="Bild1" src="https://github.com/user-attachments/assets/391fe9fc-d47c-4a21-b201-d26a262329a1">


Bild 2


<img width="390" alt="Bild2" src="https://github.com/user-attachments/assets/62a08a40-ff41-4cc7-bd8d-205ee3da93df">

**Comment by ildar170975** [comment]
Probably because you are not using exactly the card I suggested.

**Reply by thoath2** [reply]
What do you mean?

**Reply by ildar170975** [reply]
You are ether using not a stack-in-card or a custom theme with confusingly set css properties.

**Reply by thoath2** [reply]
I use the card "horizontal-stack" and the theme MacOS from HACS.
`
type: horizontal-stack
cards:
  - type: custom:tabbed-card
    options: {}
    tabs:
      - card:
          type: entities
          header: false
          title: Tagesverbrauch (Total)
          entities:
            - type: section
              label: Gesamtverbrauch
            - entity: sensor.pv_statistik_gesamtverbrauch_total_energie_tageswert
              name: Energie
              secondary_info: last-changed
            - type: section
              label: Eigenverbrauch
              align: left
            - entity: sensor.pv_statistik_eigenverbrauch_total_energie_tageswert
              name: Energie
              secondary_info: last-changed
            - entity: sensor.pv_statistik_eigenverbrauchsquote_total_energie_tageswert
              name: Anteil
            - type: section
              label: Direktverbrauch
            - entity: sensor.pv_statistik_direktverbrauch_total_energie_tageswert
              name: Energie
              secondary_info: last-changed
            - entity: >-
                sensor.pv_statistik_direktverbrauchsquote_total_energie_tageswert
              name: Anteil
            - type: section
              label: Autarkie
            - entity: sensor.pv_statistik_autarkiequote_total_energie_tageswert
              name: Anteil
        attributes:
          label: Total
      - card:
          type: entities
          header: false
          title: Tagesverbrauch (EG,DG,GEM)
          entities:
            - type: section
              label: Gesamtverbrauch
            - entity: sensor.pv_statistik_gesamtverbrauch_nur_z2_energie_tageswert
              name: Energie
              secondary_info: last-changed
            - entity: >-
                sensor.pv_statistik_gesamtverbrauchsquote_nur_z2_zu_total_energie_tageswert
              name: Anteil an Total
            - type: section
              label: Eigenverbrauch
            - entity: sensor.pv_statistik_eigenverbrauch_nur_z2_energie_tageswert
              name: Energie
              secondary_info: last-changed
            - entity: >-
                sensor.pv_statistik_eigenverbrauchsquote_nur_z2_energie_tageswert
              name: Anteil
            - type: section
              label: Direktverbrauch
            - entity: sensor.pv_statistik_direktverbrauch_nur_z2_energie_tageswert
              name: Energie
              secondary_info: last-changed
            - entity: >-
                sensor.pv_statistik_direktverbrauchsquote_nur_z2_energie_tageswert
              name: Anteil
            - type: section
              label: Autarkie
            - entity: sensor.pv_statistik_autarkiequote_nur_z2_energie_tageswert
              name: Anteil
        attributes:
          label: EG,DG,GEM
      - card:
          type: entities
          header: false
          title: Tagesverbrauch (WP+Periph.)
          entities:
            - type: section
              label: Gesamtverbrauch
            - entity: sensor.pv_statistik_gesamtverbrauch_nur_z1_energie_tageswert
              name: Energie
              secondary_info: last-changed
            - entity: >-
                sensor.pv_statistik_gesamtverbrauchsquote_nur_z1_zu_total_energie_tageswert
              name: Anteil an Total
            - type: section
              label: Eigenverbrauch
            - entity: sensor.pv_statistik_eigenverbrauch_nur_z1_energie_tageswert
              name: Energie
              secondary_info: last-changed
            - entity: >-
                sensor.pv_statistik_eigenverbrauchsquote_nur_z1_energie_tageswert
              name: Anteil
            - type: section
              label: Direktverbrauch
            - entity: sensor.pv_statistik_direktverbrauch_nur_z1_energie_tageswert
              name: Energie
              secondary_info: last-changed
            - entity: >-
                sensor.pv_statistik_direktverbrauchsquote_nur_z1_energie_tageswert
              name: Anteil
            - type: section
              label: Autarkie
            - entity: sensor.pv_statistik_autarkiequote_nur_z1_energie_tageswert
              name: Anteil
        attributes:
          label: WP+Periph.
title: Tagesverbrauch

`

**Reply by ildar170975** [reply]
1.  With my all due respect - no need to post a long code with plenty of unrelated details - it is a distraction. Just a short code with a stock stack card and the tabbed-card (without inner cards) is enough. 
2. Please use a proper formatting when posting a code. 
3. What I suggested you was a stack-in-card custom card, not a stock vertical/horizontal stack card.

**Reply by thoath2** [reply]
Hello ildar170975 ,
To your answers:
1.) You are right, it was too much code. The next time I keep the code shorter.
2.) I had formatted the code with <>-Button as a code. I have no idea why Github did not present the code properly.
3.) I have a problem with the stack-in-card. I can download this card via HACS. It is then also listed in HACS. But I can't add it with the dashboard editor, there it is missing.

**Reply by ildar170975** [reply]
2.  Place a code inside triple quotes.
3. Discussing stack-in-card is an off topic here. If it is not present in UI - then it should be added manually in yaml like plenty of custom cards.

**Reply by thoath2** [reply]
I am overwhelmed. How do I add the card manually to yaml, in which file?
And why I can't see the card not in UI? 
<img width="380" alt="Ressourcen" src="https://github.com/user-attachments/assets/a4ee4383-32d0-488c-be79-a79f5acb8269">

**Reply by ildar170975** [reply]
Discussing other custom cards in an off-topic here. Please ask in HA community.
